docker-compose.yml

```diff
@@ -33,3 +32,10 @@ services:
       - "9200:9200"
       - "9300:9300"
     tty: true
+
+  redis:
+    image: redis:latest
+    env_file: docker-env
+    ports:
+      - "6379:6379"
+    tty: true
```

Comment:
Could you add an extra line at the end of the file? :)

Reply:
Fix